### PR TITLE
feat: Modified festival page

### PR DESCRIPTION
**Pull request by Manavpatel2005**

### CODE REVIEW

File: src/Pages/Festival/Festival.ts [hunk 1]
```diff
@@ -0,0 +1,6 @@
+export interface Festival {
```
Comment (vaidehi27) [inline src/Pages/Festival/Festival.ts]: Rename this file to `types.ts`. It is not a react component and so file name should not start with a capital letter

File: package.json [hunk 2]
```diff
@@ -65,16 +66,6 @@
...
-    "husky": "^8.0.3",
-    "postcss-scss": "^4.0.6",
-    "postcss-styled": "^0.34.0",
-    "postcss-styled-syntax": "^0.4.0",
-    "react-router-dom": "^6.8.2",
-    "stylelint": "^15.3.0",
-    "stylelint-config-standard": "^31.0.0",
-    "stylelint-config-styled-components": "^0.1.1",
-    "stylelint-processor-styled-components": "^1.10.0"
+    "react-router-dom": "^6.8.2"
```
Comment (vaidehi27) [inline package.json]: why is this in dev dependency?

File: src/Components/Navbar/index.tsx [hunk 3]
```diff
@@ -1,25 +1,19 @@
...
-              AboutUs
-            </Link>
-
-            <Link className="nav-link" to="Pages/Donation">
-              Donation
-            </Link>
-            <Link className="nav-link" to="#">
-              GO SEVA
-            </Link>
+            <Link className="link" to="/Festival">
```
Comment (vaidehi27) [inline src/Components/Navbar/index.tsx]:
```suggestion
            <Link className="link" to="/festival">
```

File: src/Pages/Festival/FestivalCard.tsx [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+        ) : (
+          <>
+            <Description>
+              {festival.description.substring(0, 100)}...
+            </Description>
+            <ButtonStyle>
+              <button onClick={toggleSelected}>Read More</button>
+            </ButtonStyle>
+          </>
+        )}
```
Comment (vaidehi27) [inline src/Pages/Festival/FestivalCard.tsx]:
```suggestion
        <Description>
          {isSelected ? festival.description : festival.description.substring(0, 100)}...
        </Description>
        <ButtonStyle>
            <button onClick={toggleSelected}>{isSelected ? "Show Less" :  "Read More"}</button>
         </ButtonStyle>
```

Comment (Manavpatel2005) [reply src/Pages/Festival/FestivalCard.tsx]: changed done

File: src/Pages/Festival/FestivalCard.tsx [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+        </Heading2>
+        <Paragraph1>
+          <p>{festival.date}</p>
+        </Paragraph1>
+        <Description>
+          {isSelected
+            ? festival.description
+            : festival.description.substring(0, 100)}
+          ...
+        </Description>
```
Comment (vaidehi27) [inline src/Pages/Festival/FestivalCard.tsx]:
```suggestion
            : `${festival.description.substring(0, 100)}...`}
        </Description>
```

File: src/Pages/Festival/FestivalCard.tsx [hunk 6]
```diff
@@ -0,0 +1,46 @@
...
+
+export const FestivalCard = ({ festival }: FestivalCardProps) => {
+  const [isSelected, setIsSelected] = useState<boolean>(false);
+
+  const toggleSelected = () => {
+    setIsSelected(!isSelected);
+  };
+
+  return (
+    <div key={festival.name}>
```
Comment (vaidehi27) [inline src/Pages/Festival/FestivalCard.tsx]:
```suggestion
    <div>
```

**Review by vaidehi27 — APPROVED** [review]

**Pull request merged**